wsmaster/che-core-api-devfile/src/main/java/org/eclipse/che/api/devfile/server/Constants.java

```diff
@@ -27,13 +29,24 @@
 
   public static final String DOCKERIMAGE_TOOL_TYPE = "dockerimage";
 
+  /** Action type that should be used for commands execution. */
+  public static final String EXEC_ACTION_TYPE = "exec";
+
   /**
-   * Workspace attribute which contains comma-separated list of mappings of tool id to its name
-   * Example value:
+   * Workspace config attribute which contains comma-separated list of mappings of chePlugin tool id
+   * to its name.
+   *
+   * <p>Example value:
    *
    * <pre>
-   * eclipse/maven-jdk8:1.0.0=mvn-stack,eclipse/theia:0.0.3=theia-ide,eclipse/theia-jdtls:0.0.3=jdt.ls
+   * eclipse/maven-jdk8:1.0.0=mvn-stack,eclipse/theia-jdtls:0.0.3=jdt.ls
    * </pre>
    */
-  public static final String ALIASES_WORKSPACE_ATTRIBUTE_NAME = "toolsAliases";
+  public static final String PLUGINS_TOOLS_ALIASES_WORKSPACE_ATTRIBUTE_NAME = "pluginToolsAliases";
+
+  /** Workspace config attribute which contains cheEditor tool name. */
+  public static final String EDITOR_TOOL_ALIAS_WORKSPACE_ATTRIBUTE_NAME = "editorToolName";
+
+  /** Workspace command attributes that indicates with which tool it is associated. */
+  public static final String TOOL_NAME_COMMAND_ATTRIBUTE = "toolName";
```

Comment:
Isn't `TOOL_NAME_COMMAND_ATTRIBUTE_NAME` looks more aligned ?

Reply:
You're right. Not sure that `NAME` suffix is needed there but it looks more aligned.
Will take a look and add `_NAME` suffix here or remove in other constants in this class.